### DISCUSSION TITLE
Welcome to Flappy-Bird Discussions!

### DISCUSSION THREAD

**Discussion by moha-b** [moha-b/Flappy-Bird#16] [Q&A] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by player06** [comment, marked as answer]
what about the score bug

**Reply by moha-b** [reply]
work on it

**Comment by Drishtianand19** [comment]
i was working on your code but the audio is not working. could you tell me a way how can i add it

**Reply by moha-b** [reply]
the audio was implemented previously here 4c8b869d1bb61a10707e4dce97dc0f61113d95ea but the code is not clean in this commit 
there are more enhancements that must be done like 
> remove the game functionality to another class 

I apologize for being late. I was going to implement the audio. It will come later cuz of many things.
